desktop/hardened-desktop/admin-controls/index.md

```diff
@@ -0,0 +1,54 @@
+---
+description: admin controls for desktop
+keywords: admin controls, rootless, docker desktop, hardened desktop
+title: What is Admin Controls?
+--- 
+>Note
+>
+>Admin Controls is available to Docker Business customers only. 
+
+Admin Controls is a feature that gives admins the ability to configure certain Docker Desktop settings for their organization. 
+
+With a few lines of JSON, admins can configure Docker Desktop settings for HTTP proxies, network settings, and the Docker Engine configuration, which reduces effort spent on securing developer workflows. If admins also enable [Enhanced Container Isolation](../enhanced-container-isolation/index.md), it ensures that any configurations set with Admin Controls cannot be modified by containers.
```

Comment:
Hey @aevesdocker , could you please adjust this sentence to note that there are more settings than just proxies / network / engine